docs/content/en/integrations/parsers/file/mobsf_scorecard.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "MobSF Scorecard Scanner"
+toc_hide: true
+---
+Export a JSON file using the API, api/v1/report\_json.
```

Comment:
```suggestion
Export a JSON file using the API, api/v1/report_json.
```
Not sure if this is a typo or not

Reply:
Copied this typo from original parser, fixed )